docs/source/cli_help/github.rst

```diff
@@ -20,7 +20,9 @@ Creates, checks out and manages GitHub PRs while keeping them reflected in branc
 ``anno-prs``:
     Annotates the branches based on their corresponding GitHub PR numbers and authors.
     Any existing annotations are overwritten for the branches that have an opened PR; annotations for the other branches remain untouched.
-    Equivalent to ``git machete anno --sync-github-prs``. Doesn't modify existing ``git machete traverse`` branch qualifiers.
+    Equivalent to ``git machete anno --sync-github-prs``.
+    When the current user is **not** the owner of the PR associated with that branch, adds ``rebase=no push=no`` branch qualifiers used by ``git machete traverse``,
```

Comment:
I suppose that if the qualifiers are there already, they don't accumulate right? As in, we don't end up with `foo PR #123 rebase=no rebase=no push=no` etc.

Reply:
Judging by the implementation, they don't

Reply:
No we don't, they are added only if there are no already existing qualifiers for that branch